### PR TITLE
corrections by bfft

### PR DESCRIPTION
**Pull request by ebroecker**
Hi @BavariaBlue 

here is the next shot for my changes.

Could you please have a look on it and maybe merge?

best regards,
Eduard

### CODE REVIEW

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ TODO change activity B of LSM1
 | | LSM1 | LSM2 | LSM3 |
 |:-------------|:------------- |:---------------|:-------------|
 |**Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released hardware and software versions and variants.** | **Minimize negative impacts for customers.**|
- **Activities** | **A.** Maintenance of attack pattern catalogue **B.** Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
+ **Activities** | **A.** Maintenance of attack pattern catalogue | **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
```
Comment (BavariaBlue) [inline Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: Activities: A. --> Where there is an A activity, there also must be a B Activity for LSM1.

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md [hunk 2]
```diff
@@ -703,18 +703,18 @@ TODO add levels & remove automotive-only character
 |:-------------|:-------------|:---------------|:-------------|:------------|
 | **SCORE** | **0.0** | **0.2** | **0.5** | **1.0** |
 | **LSM1**
-| ✦Does your organization gather information about new vulnerability issues? | ??? | ??? | ??? | ??? |
-| ✦Are these issues filtered to dedicated project, used hardware and software stacks? | ??? | ??? | ??? | ??? |
-| ✦Are there project teams after final (serial) release responsible for these issues? | ??? | ??? | ??? | ??? |
+| ✦Does your organization gather information about new vulnerability issues? | NO | SOME | HALF  | MOST |
+| ✦Are these issues filtered to dedicated project, used hardware and software stacks? | NO | SOME | HALF  | MOST|
+| ✦Are there project teams after final (serial) release responsible for these issues? | NO | SOME | HALF  | MOST |
```
Comment (BavariaBlue) [inline Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:716]: "NO SOME HALF MOST" is one option to answer. There are more options in the Assessment Excel sheet, as goes:
image

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md [hunk 3]
```diff
@@ -3171,9 +3171,9 @@ Since code signing is not appropriate for everything, architects and developers
...
-| **Results** | ✦Company has improved security knowledge ✦Up to date overview of potential attack surfaces ✦c.	Always up to date catalogue of attack patterns linked with probably impacts | ✦Detailed matrix which links attack patterns to ECUs ✦Database with references from software/hardware variants of ECU and possible attack patterns ✦Detailed TODO-List / Bug-List for each product/ECU. | ✦Company has knowledge about current security risk of their products ✦Company has process and infrastructure for deploying security information and updates to end user ✦Security is guaranteed because all patches and workarounds are deployed to customers/end users|
+| **Activities** | **A.** Maintenance of attack pattern catalogue **B.** 1.	Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Operation of deployment infrastrucure **B.** Deploy  degradations/updates to customer |
```
Comment (BavariaBlue) [inline Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: typo fix: infrastructure

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md [hunk 4]
```diff
@@ -3202,91 +3199,125 @@ TODO add long description + metrics, costs, personnel, related levels
...
 ### Minimize negative impacts for customers
 
 ## Activities
-##### A. Deploy degradations to customer
-Already when starting a new ECU project, one has to clarify the communication to the end user/customer. This is essential to ensure later updates and or degradations. Therefore, an infrastructure has to be set up. This infrastructure has to be cleared with OEM also. It has to be maintained and ensured that it is "future save". Thus, it has to guarantee later updated can be deployed.
-Secondly the infrastructure is itself a high-risk channel to the ECUs in the field.
-This leads also to many efforts to be made to keep this channel secure and still working.
-Security and reliability have to be in focus.
-After in step 2 (Doing) a security update has been developed or a potential degradation is decided this has to be deployed to the ECUs out in the field. Thereby only affected ECUs should get an update. This can be dependent to Hardware and or Software version and variant. Therefore, usually at OEM side a database with software and hardware Variants have to be maintained and used for the deployment of security updates.
+##### A. Operation of deployment infrastrucure
```
Comment (BavariaBlue) [inline Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: typo fix: infrastructure

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md [hunk 5]
```diff
@@ -3171,9 +3171,9 @@ Since code signing is not appropriate for everything, architects and developers
 | | **LSM1** | **LSM2** | **LSM3** |
 |:-------------|:-------------|:---------------|:-------------|:------------|
 | **Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released Hardware and Software versions and variants.** | **Minimize negative impacts for customers.**|
```
Comment (BavariaBlue) [inline Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: Hardware and Software --> hardware and software

**Review by BavariaBlue — CHANGES_REQUESTED** [review]
Hi Eduard!

I added some comments to the pull request. Please update some of them if you feel it is necessary.

For future assessment questions, a long and short description is mandatory. Please provide these descriptions in this pull request. The location of these descriptions can be searched by the term "TODO add short" or "TODO add long" in SAMM-1.5.1.md.

Btw., great job by branching into BFFT and pull requesting into master back again 👍

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md (hunk 5 above)
Comment (ebroecker) [reply Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md (hunk 4 above)
Comment (ebroecker) [reply Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md (hunk 3 above)
Comment (ebroecker) [reply Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md (hunk 1 above)
Comment (ebroecker) [reply Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md]: should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08

File: Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md (hunk 2 above)
Comment (ebroecker) [reply Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md:716]: I'll rethink these points

**Comment by ebroecker** [comment]
@BavariaBlue 
Thanks for reviewing,
I hopefully fixed the easy to fix typos.

I'll rethink the questions/answers ~~and for sure I have to provide the  long and short description~~ the next days.

**Pull request merged**